### DISCUSSION TITLE
How to disable 3D Touch / Habtics on WebView for a more native feeling?

### DISCUSSION THREAD

**Discussion by mariusbolik** [ionic-team/capacitor#4378] [Q&A]
Hello together,

I have the following issue: No matter which element you hold pressed, haptic feedback always appears. This makes the user notice that the app is running in a webview. I would like to disable this behavior. I tried setting `allowsLinkPreview` to false in my capacitor.config.json, but it didn't work. Is there any possibility to remove the haptic feedback from the webview?

A big thanks to the capacitor team for all the great work!

Best regards,
Marius

**Comment by vojto** [comment]
https://github.com/ionic-team/capacitor/discussions/3208
